### PR TITLE
Correct pipe Test

### PR DESCRIPTION
**Pull request by g-boy05**
I coorect my PB and i add the git configuration

### CODE REVIEW

**Comment by deepu105** [comment]
@g-boy05 did you test this locally? I dont this this will compile can you please test locally and update the PR

**Comment by g-boy05** [comment]
I have already tested the branch master in generator-jhipster and it was compiled successfully.��I don't understand why you told me to move on this project ?

**Comment by deepu105** [comment]
Coz the files where moved from generator to this library


Thanks & Regards,
Deepu

**Comment by g-boy05** [comment]
Ok i tested on local it's compile successfully.
Thanks
Zakaria

**Comment by g-boy05** [comment]
thank you pascal
you can check now i corrected the all errors .

**Comment by pascalgrimaud** [comment]
@g-boy05 : sorry but Travis still failed

**Comment by deepu105** [comment]
You have tslint errors and test failures can you please run `npm run lint && npm test` on your local machine on this project and make sure it passes first

**Comment by g-boy05** [comment]
The all test are failed to run, failed to run npm run lint !!

**Comment by g-boy05** [comment]
@deepu105:  you can check now

**Comment by deepu105** [comment]
You still have tslint errors

File: tests/pipe/order-by.pipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {OrderByPipe} from '../../src/pipe/order-by.pipe';
+
+describe('orderBy Tests', () => {
```
Comment (deepu105) [inline tests/pipe/order-by.pipe.spec.ts]: OrderByPipe tests

**Pull request merged**